### ISSUE TITLE
Too much root elements in FrankConfig.xsd

### ISSUE THREAD

**Issue by jacodg** [ibissource/frank-doc#92]
According to the current FrankConfig.xsd the following root elements are allowed (screenshot of design view in Eclipse):

![image](https://user-images.githubusercontent.com/2970084/159686598-62d8ac1e-d1bf-4c1c-8c22-8b3d68957ffa.png)

While the old xsd shows the only three allowed root elements:

![image](https://user-images.githubusercontent.com/2970084/159686814-f3c41c47-e065-4c4f-92af-45d8c8df0a82.png)

**Comment by gvanbrakel** [comment]
I think this should be solved by referring to types in stead of to elements.
E.g. Adapter now has `<xs:element ref="Receiver">`, this should probably be something like  `<xs:element name="Receiver" type="ReceiverType"`

**Comment by mhdirkse** [comment]
I remember that we introduced all these root elements deliberately. The reason we introduced them was that the structure with types, declared groups and cumulative groups is more complicated then having plain elements. For elements that are not part of a type based on an interface (e.g. Configuration, Forward), using types and groups does not save space. We decided to put these as plain elements to the start of the XSDs to make them more readable.

My question is: now that we do not want plain elements, shall I try to do everything the same, focusing on simplifying the Java code? Or do we still want to avoid cumulative and declared groups where we now have plain elements?
